package.json

```diff
@@ -4,7 +4,7 @@
   "main": "dist/lib.js",
   "license": "MIT",
   "scripts": {
-    "test": "",
+    "test": "mocha -r ./babel-register.js 'src/**/*.spec.ts'",
```

Comment:
Jest automatically handles splitting tests across processes, collecting and aggregating coverage. Also has nice built-in watch options. I think this might be a better choice for the project.

Reply:
Agreed.